src/app/functions/server/featureFlags.test.ts

```diff
@@ -0,0 +1,58 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// import { it, expect, jest } from "@jest/globals";
+
+import { getFeatureFlagByName } from "../../../db/tables/featureFlags";
+jest.mock("../../../db/tables/featureFlags");
+
+jest.mock("next-auth", () => ({
+  getServerSession: jest.fn(() => {
+    return { user: { email: "test@example.com" } };
+  }),
+}));
+
+jest.mock("../../api/utils/auth", () => ({
+  authOptions: jest.fn(),
+}));
+
+jest.mock("../../../db/tables/featureFlags", () => ({
+  getFeatureFlagByName: jest.fn(),
+}));
+
+import { isFlagEnabled } from "./featureFlags";
+
+beforeEach(() => {
+  jest.resetModules();
+});
+
```

Comment:
Hmm, I'd swear that I'd enabled `resetMocks` and `resetModules` by default when I set up Jest, but [apparently not](https://github.com/mozilla/blurts-server/blob/698e67a462a3a30ea46c0d243b166d0f0aadce73/jest.config.cjs#L126-L130). Either way, I think it's preferable to enable that globally rather than setting that for each (or worse, some) individual test, for predictability. Tests should never rely on the results from previous tests anyway.